pkgs/misc/emulators/retroarch/cores.nix

```diff
@@ -403,11 +404,7 @@ in
     description = "Port of FBNeo to libretro";
     license = "Non-commercial";
     makefile = "Makefile";
-    postPatch = ''
-      sed -i -e 's:-Wall:-Wall -Wno-format-security:g' src/burner/libretro/Makefile
-    '';
     preBuild = "cd src/burner/libretro";
-    makeFlags = [ "USE_EXPERIMENTAL_FLAGS=1" ];
```

Comment:
Didn't find any reference in the source code about this flag.

Reply:
The slow platforms that upstream enable are either a Funkey S or a Classic NES/SNES that should be much slower than any platform we support.